### PR TITLE
Here are my tests for average

### PR DESCRIPTION
**Pull request by hari-md**

### CODE REVIEW

File: Summator.UnitTests/SummatorTests.cs [hunk 1]
```diff
@@ -13,6 +14,7 @@ public void Test_Summator_SumTwoPositiveNumbers()
```
Comment (YuliaStarostenko) [inline Summator.UnitTests/SummatorTests.cs:14]: 1.Change the rows order:
//Arrange 
var nums...
var expected
//Act( actual execution of the method under testing)
var actual ...
//Assert....
2.Between these "phases" should be an empty line
3. Change names of the variables. They should be meaningful. expected and actual are not very describing. Smth like actualResult or actualSum...

File: Summator.UnitTests/SummatorTests.cs [hunk 2]
```diff
@@ -4,6 +4,7 @@ namespace Summator.UnitTests
 {
     public class SummatorTests
     {
+        //Tests for Summator:
         [Test]
         public void Test_Summator_SumTwoPositiveNumbers()
```
Comment (YuliaStarostenko) [inline Summator.UnitTests/SummatorTests.cs:9]: pls change test name. Refer to what you test. Like TestSum_With_TwoPositive...

File: Summator.UnitTests/SummatorTests.cs (hunk 1 above)
Comment (YuliaStarostenko) [reply Summator.UnitTests/SummatorTests.cs:14]: These two above comments are for all tests.
Pls revise and write me again)

File: Summator.UnitTests/SummatorTests.cs (hunk 2 above)
Comment (hari-md) [reply Summator.UnitTests/SummatorTests.cs:9]: ok

**Pull request merged**